src/Entity/Transformer.php

```diff
@@ -100,8 +100,8 @@ public function create(PageInterface $page)
         $content['meta_keyword'] = $page->getMetaKeyword();
         $content['template_code'] = $page->getTemplateCode();
         $content['request_method'] = $page->getRequestMethod();
-        $content['created_at'] = $page->getCreatedAt()->format('U');
-        $content['updated_at'] = $page->getUpdatedAt()->format('U');
+        $content['created_at'] = (int) $page->getCreatedAt()->format('U');
+        $content['updated_at'] = (int) $page->getUpdatedAt()->format('U');
```

Comment:
psalm does not report as numeric-string, and even if it does, it makes sense to me to typehint as int|null, since it is better to store it correctly instead of having to convert to int on the decode.